content/articles/convert-html-to-pdf-with-docraptor/index.md

```diff
@@ -0,0 +1,175 @@
+### Introduction
+It is easy for anyone to deliver PDFs because it is a standard forat for document presentation. Everything is given as a PDF now, including requests, receipts, and appropriation reports. However, the planned duration of PDF can be inconvenient if you need to create a large number of PDFs quickly.
+
+DocRaptor is an HTML-to-PDF API that greatly enhances the current situation in addition to having a PHP framework for simple joining. This article will guide the reader on how to use DocRaptor to converts HTML into a PDF.
+
+### Table of content
+- [Installation](#installation)
+- [Authentication](#authentication)
+- [Add HTML or a URL](#add-html-or-a-url)
+- [API options](#api-options)
+- [Get documentation](#get-documentation)
+- [Conclusion](#conclusion)
+  - [Further activity reading](#further-activity-reading)
+  
+### Installation
+Download the most recent version of DocRaptor to your computer and unzip it into your hard drive. You have two options for for doing this the first is using an author and the second is when working alone.
+
+If you are collaborating with an author, you will need to mention their name in the purchase line of your order confirmation.
+

```

Comment:
Elaborate on how this is done clearly. I am trying to understand where the author is coming from. Let the reader know the complete steps of installation. Remember we are writing for beginners as well.